From 37a51b85a338bc2e24411e1d1caa62a8e2231bd8 Mon Sep 17 00:00:00 2001
From: Vaclav Slavik <vslavik@gmail.com>
Date: Tue, 5 Feb 2013 20:59:42 +0100
Subject: [PATCH] Compilation fix for wxWidgets 2.8.

Fixes #513.
---
 src/catalog.cpp | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/catalog.cpp b/src/catalog.cpp
index a4fbfa9..ccb6c71 100644
--- a/src/catalog.cpp
+++ b/src/catalog.cpp
@@ -1151,8 +1151,12 @@ inline bool CanEncodeStringToCharset(const wxString& s, wxMBConv& conv)
 {
     if (s.empty())
         return true;
+#if wxCHECK_VERSION(2,9,0)
     wxCharBuffer converted(s.mb_str(conv));
     if ( converted.length() == 0 )
+#else
+    if ( wxStrlen(s.mb_str(conv)) == 0 )
+#endif
         return false;
     return true;
 }
-- 
1.8.1.6

